src/main/java/gov/usgs/earthquake/origin/OriginIndexerModule.java

```diff
@@ -92,16 +121,116 @@ public void setPlacesService(GeoservePlaces geoservePlaces) {
     this.geoservePlaces = geoservePlaces;
   }
 
+  /**
+   * Set the geoserveRegions to be used for subsequent calls to GeoServe regions
+   * endpoint.
+   *
+   * @param geoserveRegions The GeoserveRegions to use
+   */
+  public void setRegionsService(GeoserveRegionsService geoserveRegions) {
+    this.geoserveRegions = geoserveRegions;
+  }
+
+  /**
+   * Set the distance threshold to prefer fe region over nearst place
+   * in the event title
+   *
+   * @param threshold The distance threshold to use
+   */
+  public void setDistanceThreshold(Double threshold) {
+    this.distanceThreshold = threshold;
+  }
+
   @Override
   public void configure(Config config) throws Exception {
-    String endpointUrl = config.getProperty(ENDPOINT_URL_PROPERTY, GeoservePlacesService.DEFAULT_ENDPOINT_URL);
+    String endpointUrl = config.getProperty(
+        ENDPOINT_URL_PROPERTY,
+        GeoservePlacesService.DEFAULT_ENDPOINT_URL
+    );
     int connectTimeout = Integer.parseInt(
-        config.getProperty(CONNECT_TIMEOUT_PROPERTY, Integer.toString(GeoservePlacesService.DEFAULT_CONNECT_TIMEOUT)));
+        config.getProperty(
+            CONNECT_TIMEOUT_PROPERTY,
+            Integer.toString(GeoservePlacesService.DEFAULT_CONNECT_TIMEOUT)
+        )
+    );
     int readTimeout = Integer.parseInt(
-        config.getProperty(READ_TIMEOUT_PROPERTY, Integer.toString(GeoservePlacesService.DEFAULT_READ_TIMEOUT)));
+        config.getProperty(
+            READ_TIMEOUT_PROPERTY,
+            Integer.toString(GeoservePlacesService.DEFAULT_READ_TIMEOUT)
+        )
+    );
+    this.distanceThreshold = Double.parseDouble(
+        config.getProperty(
+            GEOSERVE_DISTANCE_THRESHOLD_PROPERTY,
+            DEFAULT_GEOSERVE_DISTANCE_THRESHOLD.toString()
+        )
+    );
 
     LOGGER.config(String.format("[%s] GeoservePlacesService(%s, %d, %d)", this.getName(), endpointUrl, connectTimeout,
         readTimeout));
     this.setPlacesService(new GeoservePlacesService(endpointUrl, connectTimeout, readTimeout));
```

Comment:
I think you also need to configure the GeoserveRegionsService here?

Reply:
Updated.